package.json

```diff
@@ -103,15 +104,24 @@
     "stylelint-config-concentric": "^2.0.2",
     "stylelint-declaration-use-variable": "1.7.0",
     "stylelint-scss": "3.2.0",
+    "timekeeper": "1.0.0",
     "tosource-polyfill": "^0.3.1",
+    "typeahead": "twitter/typeahead.js#v0.11.1",
     "uuid": "^3.0.0",
+    "webrtc-adapter": "^3.1.3",
     "worker-pouch": "2.1.0"
   },
   "dependencies": {
+    "broccoli-funnel": "^2.0.1",
```

Comment:
Why were so many packages moved to `dependencies` and not in the `devDeps`? I'd say we deliver our assets which already contain everything bundled; probably it's not worth to explicitly require some packages to be installed afterwards. Correct me if I'm wrong.

Reply:
TBH, it's because I can never remember the difference in commands between yarn add and the `npm i --save-dev`.

I assume the electron stuff is a requirement there for how the electron builds happen. I moved everything up to dependencies minus the electron stuff.